core/src/main/kotlin/core/io/StandardMid.kt

```diff
@@ -25,11 +24,8 @@ object StandardMid {
 
     suspend fun parse(file: File): Project {
         val midi = Mid.parseMidi(file)
-        if (midi == false) {
-            throw IllegalFileException.IllegalMidiFile()
```

Comment:
How does the new library handle unexpected input?
If it throws an error, we could catch it and rethrow the `IllegalFileException.IllegalMidiFile()` with the original error as the cause (passed by constructor)

Reply:
It looks like it throws string (not Error). I changed to catch `dynamic` so that the error will be handled